deploy/lib/server_config.rb

```diff
@@ -1362,6 +1379,41 @@ def settings
 
 private
 
+  def filter_to_newer_files(files, options)
+    logger.info "Filtering to files which are newer locally than on the server"
+
+    if @server_version < 6
+      raise ExitException.new("Can only filter files on MarkLogic 6 and later")
+    end
+
+    uris = files.map { |f| xcc.build_target_uri(f, options) }
+    stamps_db = get_db_timestamps(uris)
+    stamps_local = files.map { |file_uri| File.mtime(file_uri).getgm.iso8601(5).tr('Z','') }
```

Comment:
I've now pushed a change to ensure that both the server timestamps and the local ones are in UTC. One of the joys of being UK-based is that it's easy to make timezone screw ups without noticing since half the year we are on UTC...